apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1193,18 +1199,20 @@ export class WorkshopForm extends React.Component {
                       aria-labelledby="dropdownMenuButton"
                     >
                       <ul style={styles.listItems}>
-                        {ALL_PL_TOPICS.map(label => (
+                        {Object.values(ALL_PL_TOPICS).map(object => (
```

Comment:
Nit: could `object` be renamed to `topic`?

Reply:
Good call- more clear. Updated!